roles/custom/matrix-nginx-proxy/templates/nginx/nginx.conf.j2

```diff
@@ -39,7 +39,18 @@ http {
 
 	{% if matrix_nginx_proxy_access_log_enabled %}
 	access_log /var/log/nginx/access.log main;
-	{% else %}
+	{% endif %}
+
+	{% if matrix_prometheus_nginxlog_exporter_enabled %}
```

Comment:
We try to avoid referencing variables from other roles like this.

This should probably use `matrix_nginx_proxy_*` variables, which are defined in terms of `matrix_prometheus_nginxlog_exporter` variables via `group_vars/matrix_servers`.

Example `roles/custom/matrix-nginx-proxy/default/main.yml`:

```yaml
matrix_nginx_proxy_access_log_syslog_integration_enabled: false

matrix_nginx_proxy_access_log_syslog_integration_server: ''
matrix_nginx_proxy_access_log_syslog_integration_tag: 'matrix_nginx_proxy'
```

If `6514` is some port number, maybe we need (`matrix_nginx_proxy_access_log_syslog_integration_port` as well).

----

Then, from the `matrix-nginx-proxy` section in `group_vars/matrix_servers`, we can have something like this:

```yaml
matrix_nginx_proxy_access_log_syslog_integration_enabled: "{{ matrix_prometheus_nginxlog_exporter_enabled }}"

matrix_nginx_proxy_access_log_syslog_integration_server: "{{ matrix_prometheus_nginxlog_exporter_container_url if matrix_prometheus_nginxlog_exporter_enabled else '' }}"
```